Problem1/tc_even_numbers_sum.rb

```diff
@@ -0,0 +1,48 @@
+# File:  tc_even_numbers_sum.rb
+
+require_relative "even_numbers_sum.rb"
+require "test/unit"
+
+class TestEvenNumbersSum < Test::Unit::TestCase
+	
+	def test_empty
+		
+		even_numbers_sum = EvenNumbersSum.new
+		numbers = Array[]
+		
+		sum_of_even_numbers = even_numbers_sum.get_sum_of_even_numbers(numbers)		
+	
+		assert_equal(0, sum_of_even_numbers)
+	end
+	
+	def test_even_numbers
+		
+		even_numbers_sum = EvenNumbersSum.new
+		numbers = Array[10,22,32,40,82]
+		
+		sum_of_even_numbers = even_numbers_sum.get_sum_of_even_numbers(numbers)		
+	
+		assert_equal(186, sum_of_even_numbers)
+	end
+
+	
+	def test_no_even_numbers
+		
+		even_numbers_sum = EvenNumbersSum.new
+		numbers = Array[5,15,7,13,11]
+		
+		sum_of_even_numbers = even_numbers_sum.get_sum_of_even_numbers(numbers)		
+	
+		assert_equal(0, sum_of_even_numbers)
+	end
+	
+	def test_combined_even_numbers
+		
+		even_numbers_sum = EvenNumbersSum.new
+		numbers = Array[5,10,40,13,11,22,82]
+		
+		sum_of_even_numbers = even_numbers_sum.get_sum_of_even_numbers(numbers)		
+	
+		assert_equal(154, sum_of_even_numbers)
+	end
+end
```

Comment:
I think these specs would look much better in general if you used rspec/mini-test which are the most used frameworks in ruby for unit testing. In the future when deciding whether to use a certain framework/library I would recommend to take into account whether it is: stable, properly supported, with an active community, open/closed source.